README.qmd

```diff
@@ -9,13 +9,63 @@ format: gfm
+
+4. [Post-processing](https://www.viash-hub.com/packages/rnaseq/v0.2.0/components/workflows/post_processing): This is a workflow for duplicate read marking (picard MarkDuplicates), transcript assembly and quantification (StringTie), and creation of bigWig coverage files.
+
+5. [Pseudo alignment and quantification](https://www.viash-hub.com/packages/rnaseq/v0.2.0/components/workflows/pseudo_alignment_and_quant): This is a workflow for performing pseudo alignment and transcript quantification using Salmon or Kallisto. 
+
+6. [Final QC](https://www.viash-hub.com/packages/rnaseq/v0.2.0/components/workflows/quality_control): This is a workflow for performing extensive quality control (RSeQC, dupRadar, Qualimap, Preseq, DESeq2, featureCounts). It presents QC for raw reads, alignments, gene biotype, sample similarity, and strand specificity (MultiQC).
+
+### Executing individual workflow steps
+
+At the moment, this pipeline makes use of the following components from [biobox](hhttps://www.viash-hub.com/packages/biobox/latest):
+
+* [`gffread`](https://www.viash-hub.com/packages/biobox/v0.3.0/components/gffread)
+* [`star/star_genome_generate`](https://www.viash-hub.com/packages/biobox/v0.3.0/components/star/star_genome_generate)
+* [`star/star_align_reads`](https://www.viash-hub.com/packages/biobox/v0.3.0/components/star/star_align_reads)
+* [`salmon/salmon_index`](https://www.viash-hub.com/packages/biobox/v0.3.0/components/salmon/salmon_index)
+* [`salmon/salmon_quant`](https://www.viash-hub.com/packages/biobox/v0.3.0/components/salmon/salmon_quant)
+* [`featurecounts`](https://www.viash-hub.com/packages/biobox/v0.3.0/components/featurecounts)
+* [`samtools/samtools_sort`](https://www.viash-hub.com/packages/biobox/v0.3.0/components/samtools/samtools_sort)
+* [`samtools/samtools_index`](https://www.viash-hub.com/packages/biobox/v0.3.0/components/samtools/samtools_index)
+* [`samtools/samtools_stats`](https://www.viash-hub.com/packages/biobox/v0.3.0/components/samtools/samtools_stats)
+* [`samtools/samtools_flagstat`](https://www.viash-hub.com/packages/biobox/v0.3.0/components/samtools/samtools_flagstat)
+* [`samtools/samtools_idxstats`](https://www.viash-hub.com/packages/biobox/v0.3.0/components/samtools/samtools_idxstats)
+* [`multiqc`](https://www.viash-hub.com/packages/biobox/v0.3.0/components/multiqc) (work in progress - updating `assets/multiqc_config.yaml`)
+* [`fastp`](https://www.viash-hub.com/packages/biobox/v0.3.0/components/fastp) (work in progress)
+* [`rsem/rsem_prepare_reference`](https://www.viash-hub.com/packages/biobox/v0.3.0/components/rsem/rsem_prepare_reference) (work in progress)
+* [`rsem/rsem_calculate_expression`](https://www.viash-hub.com/packages/biobox/v0.3.0/components/rsem/rsem_calculate_expression) (work in progress)
+
+## Example usage
```

Comment:
Maybe a list of options (CLI, nextflow executable and seqera cloud) can be shown here, so that the reader knows that there are different options for executors before needing to scroll down? My first instinct was that there was only the nextflow executable option because that one came first